### PR TITLE
Added getOverviewMap getter method in overviewmapcontrol.js

### PR DESCRIPTION
**Pull request by bogdanvaduva**

### CODE REVIEW

**Comment by adube** [comment]
Please, see the Travis report and fix the issues raised.

**Comment by adube** [comment]
Would anyone any core developer review and merge this PR, please ?  Thank you very much.

File: src/ol/control/overviewmapcontrol.js [hunk 1]
```diff
@@ -528,3 +528,13 @@ ol.control.OverviewMap.prototype.setCollapsed = function(collapsed) {
 ol.control.OverviewMap.prototype.getCollapsed = function() {
   return this.collapsed_;
 };
+
+
+/**
+ * Return the overview map.
+ * @return {ol.Map} Overview map.
+ * @api experimental
```
Comment (ahocevar) [inline src/ol/control/overviewmapcontrol.js]: Shound be just `@api`, not `@api experimental`.

**Comment by ahocevar** [comment]
@bogdanvaduva I added one inline comment. Would be great if you could address that. Also, please sign a [CLA](https://github.com/openlayers/cla) so we can merge this

**Comment by marcjansen** [comment]
Ping @ahocevar

**Comment by ahocevar** [comment]
Thanks @bogdanvaduva.

**Comment by bogdanvaduva** [comment]
Welcome

**Pull request merged**